docs/.vitepress/config.js

```diff
@@ -641,7 +641,7 @@ export default defineConfig({
             { text: 'Text', link: '/components/text/' },
             { text: 'Text area', link: '/components/textarea/' },
             { text: 'Text field', link: '/components/textfield/' },
-            { text: 'Toast', link: '/components/toast/' },
+            { text: 'Toast <span class="beta-badge">Beta</span>', link: '/components/toast/' },
```

Comment:
Could we maybe try to keep all this in the data.json  statuses like this should not be hardcoded all over the place Imo. We are even changing the json file to Released (beta) in this same operation :)

Reply:
This doesnt really have anything to do with this pr, but more an overall observation.